Automatic Validation ended with:
> 2025-04-25T09:19:50.7711959Z ##[error] Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\d\dRofus\dRofus\EXE\2.14.20', expected 'manifests\d\dRofus\dRofus\EXE\2.14.20.370'
 2025-04-25T09:19:50.7719301Z ##[error] Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\d\dRofus\dRofus\EXE\2.14.20', expected 'manifests\d\dRofus\dRofus\EXE\2.14.20.370'

(Automated response - build 1048.)

--- a/manifests/d/dRofus/dRofus/EXE/2.14.20/dRofus.dRofus.EXE.installer.yaml
+++ b/manifests/d/dRofus/dRofus/EXE/2.14.20/dRofus.dRofus.EXE.installer.yaml
@@ -2,7 +2,7 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
 
 PackageIdentifier: dRofus.dRofus.EXE
-PackageVersion: 2.14.20.370
+PackageVersion: 2.14.20
 InstallerType: exe # Squirrel
 Scope: user
 InstallModes:

--- a/manifests/d/dRofus/dRofus/EXE/2.14.20/dRofus.dRofus.EXE.locale.en-US.yaml
+++ b/manifests/d/dRofus/dRofus/EXE/2.14.20/dRofus.dRofus.EXE.locale.en-US.yaml
@@ -2,7 +2,7 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.10.0.schema.json
 
 PackageIdentifier: dRofus.dRofus.EXE
-PackageVersion: 2.14.20.370
+PackageVersion: 2.14.20
 PackageLocale: en-US
 Publisher: dRofus AS
 PublisherUrl: https://www.drofus.com/

--- a/manifests/d/dRofus/dRofus/EXE/2.14.20/dRofus.dRofus.EXE.locale.zh-CN.yaml
+++ b/manifests/d/dRofus/dRofus/EXE/2.14.20/dRofus.dRofus.EXE.locale.zh-CN.yaml
@@ -2,7 +2,7 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.10.0.schema.json
 
 PackageIdentifier: dRofus.dRofus.EXE
-PackageVersion: 2.14.20.370
+PackageVersion: 2.14.20
 PackageLocale: zh-CN
 # Publisher:
 # PublisherUrl:

--- a/manifests/d/dRofus/dRofus/EXE/2.14.20/dRofus.dRofus.EXE.yaml
+++ b/manifests/d/dRofus/dRofus/EXE/2.14.20/dRofus.dRofus.EXE.yaml
@@ -2,7 +2,7 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.10.0.schema.json
 
 PackageIdentifier: dRofus.dRofus.EXE
-PackageVersion: 2.14.20.370
+PackageVersion: 2.14.20
 DefaultLocale: en-US
 ManifestType: version
 ManifestVersion: 1.10.0

